.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+# Builds and pushes a docker image on a new tag.
+#
+# Uses the action:
+# https://github.com/marketplace/actions/build-and-push-docker-images
+name: ci
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Get the version
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/tags/}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: massopencloud/horizon-onboarding:${{ steps.vars.outputs.tag }}
```

Comment:
If you make the image name a repository secret, you can re-use this workflow verbatim in other repositories.

Reply:
Good idea. Will do.